examples/billing_usage.py

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright (c) 2025 NSONE, Inc.
+#
+# License under The MIT License (MIT). See LICENSE in project root.
+#
+
+from ns1 import NS1
+
+# NS1 will use config in ~/.nsone by default
+api = NS1()
+
+# to specify an apikey here instead, use:
+
+# from ns1 import Config
+# config = Config()
+# config.createFromAPIKey('<<CLEARTEXT API KEY>>')
+# api = NS1(config=config)
+
+config = api.config
+
+############################
+# GET BILLING USAGE LIMITS #
+############################
+
+limits = api.billing_usage().getLimits(fromUnix=1738368000, toUnix=1740614400)
```

Comment:
Could we use something less cryptic than these numbers?
```suggestion
EPOCH_FROM = int(datetime.datetime.fromisoformat("2025-02-01 00:00:00").strftime("%s"))
EPOCH_TO = int(datetime.datetime.fromisoformat("2025-02-27 00:00:00").strftime("%s"))

limits = api.billing_usage().getLimits(fromUnix=epoch_from, toUnix=epoch_to)
```

We can use these values everywhere in the file, since they seem to be the same.